f2/src/forms/ContactInfoForm.js

```diff
@@ -45,10 +45,7 @@ export const ContactInfoForm = ({ onSubmit }) => {
   return (
     <React.Fragment>
       {false ? ( // mark ids for lingui
-        <div>
-          <Trans id="contactinfoForm.email.warning" />
-          <Trans id="contactinfoForm.phone.warning" />
-        </div>
+        <div></div>
```

Comment:
We can 🔥 this whole `{false .... }` section if there's nothing left to mark for lingui